apps/dialtone-documentation/docs/components/split-button.md

```diff
@@ -20,6 +21,31 @@ storybook: https://dialtone.dialpad.com/vue/?path=/story/components-split-button
   </dt-split-button>
 </code-well-header>
 
+## Usage
+
+This dual-functionality allows for surfacing variations of the default action. It conserves space in the interface and reduces cognitive load. While versatile, they should be used judiciously to avoild ovewhelming users or cluttering the UI.
```

Comment:
```suggestion
This dual-functionality allows for surfacing variations of the default action. It conserves space in the interface and reduces cognitive load. While versatile, they should be used judiciously to avoid overwhelming users or cluttering the UI.
```